### DISCUSSION TITLE
Cannot find chrome on Github Actions using self hosted runner

### DISCUSSION THREAD

**Discussion by maxnowack** [GoogleChrome/lighthouse-ci#732] [Q&A]
I've setup lighthouse-ci on Github Actions according to the Gettings Started Guide. I'm using self hosted runners in my kubernetes cluster for Github Actions.
Now I'm getting an error, that the Chrome installation was not found:
![image](https://user-images.githubusercontent.com/186073/145173515-da8b23a8-beca-446b-98df-0115e776fde9.png)

Do I have to install chrome/chromium somewhere?

**Comment by enriquepalaciosv** [comment]
@maxnowack Did you manage to solve this issue? I'm facing something similar.

**Reply by maxnowack** [reply]
No. Sadly not 😕
